src/orders-v2/order-header.tsx

```diff
@@ -0,0 +1,85 @@
+import { channelUrl } from "@dashboard/channels/urls";
+import { TopNav } from "@dashboard/components/AppLayout";
+import { OrderStatus } from "@dashboard/graphql";
+import { transformOrderStatus } from "@dashboard/misc";
+import { Box, Button, Text } from "@saleor/macaw-ui-next";
+import { Package } from "lucide-react";
+import moment from "moment-timezone";
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { StatusPill } from "./status-pill";
+import { UnderlineLink } from "./underline-link";
+
+interface Props {
+  status: OrderStatus;
+  orderNumber: string;
+  created: string;
+  channel: {
+    id: string;
+    name: string;
+  };
+}
+
+export const OrderHeader = ({ status, orderNumber, created, channel }: Props) => {
+  const intl = useIntl();
+
+  const { localized, status: orderStatus } = transformOrderStatus(status, intl);
+
+  const formattedDate = moment(created).format("Do MMM, YY HH:mm");
```

Comment:
Why don't you use Intl? We should probably deprecate moment.js

Reply:
I will make sure that other `Date` components are deprecated as well + moment.js as dep in eslint